README.md

```diff
@@ -6,6 +6,12 @@
 docker-compose up
 ```
 
+If you don't run Docker natively (like on macOS) you need to overwrite `HOST_DEMO_DIR`
```

Comment:
This is not macOS or docker specific. Please explain that it is absolute path to /var/demo directory of the project.

Something like "If your project source code is not in `/var/www/getrector.org`, you need to set absolute path to ...."